### ISSUE TITLE
NYT bypass broken

### ISSUE THREAD

**Issue by sokolq55** [nextgens/anti-paywall#113]
New York Times no longer works. Can still get through in incognito. Thanks.

**Comment by SpecificHat** [comment]
Blocking cookies from et.nytimes.com and meter-svc.nytimes.com seems to do the job.

**Comment by frankzhangming** [comment]
met same problem.

**Comment by sokolq55** [comment]
> Blocking cookies from et.nytimes.com and meter-svc.nytimes.com seems to do the job.

Thank you @SpecificHat - blocking cookies from those two did the trick. Do you know of a similar workaround for washingtonpost? Or something that might apply more universally to different paywalls? Thanks!

**Comment by rpodric** [comment]
A best practice in general on such sites is to set each to clear cookies (and maybe site data) upon exit.

**Comment by frankzhangming** [comment]
> > Blocking cookies from et.nytimes.com and meter-svc.nytimes.com seems to do the job.
> 
> Thank you @SpecificHat - blocking cookies from those two did the trick. Do you know of a similar workaround for washingtonpost? Or something that might apply more universally to different paywalls? Thanks!

hi,why I still cannot bypass the paywall after I have blocked cookies from et.nytimes.com and meter-svc.nytimes.com?

**Comment by rpodric** [comment]
Clearing the cookies is key. Blocking them won't do that.

**Comment by frankzhangming** [comment]
thanks,I will try again.
rpodric <notifications@github.com> 于 2019年2月17日周日 11:33写道：

**Comment by frankzhangming** [comment]
oh,hope for a new way.it;s not very convinient,but still thanks.

**Comment by rpodric** [comment]
I meant manually clear all cookies related to nytimes.com just once. Then put in the blocks (which I don't even do, but some above have recommended it).

For good measure, I have Chrome set to automatically delete cookies related to nytimes.com upon closing the browser, which I do each night anyway. If you don't close your browser that often, the block would be a lot more important.

**Comment by frankzhangming** [comment]
I have Chrome set to automatically delete cookies related to nytimes.com
upon closing the browser,but useless except manually clear all cookies
every time,very confusing.

rpodric <notifications@github.com> 于2019年2月18日周一 上午12:52写道：

**Comment by rpodric** [comment]
That doesn't make sense, since even without the extension you'd be given X number (something like 10). It doesn't sound like it's actually clearing. And what do you mean by "all" (if it's cookies for all sites, that's not needed)?  I assume you're using the recommended format in the "Clear on exit" section?

`[*.]nytimes.com`

**Comment by frankzhangming** [comment]
yes, I am using the recommended format in the "Clear on exit" section,is
there another way? Thanks.

rpodric <notifications@github.com> 于 2019年2月18日周一 09:57写道：

**Comment by rpodric** [comment]
Having that set and exiting the browser should be exactly the same as what you're doing manually, though you weren't specific about what you were deleting manually. You should test to see if the cookies are actually being deleted.

**Comment by frankzhangming** [comment]
I deleted all cookies related with newyorktimes. com, and exited. But when
I opened Chrome again, the cookies appeared again.

rpodric <notifications@github.com> 于 2019年2月18日周一 12:59写道：

**Comment by rpodric** [comment]
Nytimes.com. They will reappear if you revisit that site, but I assume you mean open Chrome to some other site and then examine cookies.

Anyway, you have a browser problem--I doubt that any cookies are deleting for you then no matter what you set. You might want to take it up from that standpoint (why don't cookies for any site that I set delete on exit) in a Chrome forum, or perhaps try Firefox.
